activesupport/lib/active_support/duration/iso8601_parser.rb

```diff
@@ -0,0 +1,122 @@
+require 'strscan'
+
+module ActiveSupport
+  class Duration
+    # Parses a string formatted according to ISO 8601 Duration into the hash .
+    #
+    # See http://en.wikipedia.org/wiki/ISO_8601#Durations
```

Comment:
Actually this should be wrapped in as a markdown link.
`See [ISO 8601](http://en.wikipedia.org/wiki/ISO_8601#Durations) for more information.`